scripts/install.sh

```diff
@@ -206,6 +216,21 @@ while [ "$1" != "" ]; do
         -r | --region )                 shift
                                         region=$1
                                         ;;
+        llt | --lambdaLatencyThreshold )         shift
```

Comment:
These abbreviated CLI options (llt, aset, abet, let, delet) really should be removed entirely, because they are so unconventional, and not used.  The long (double dash --) options should suffice.